src/components/ebay-section-title/index.marko

```diff
@@ -41,22 +44,6 @@ static var ignoredAttributes = [
             </span>
         </if>
     </div>
-    <!-- CTA -->
```

Comment:
Do we need to retain this for backwards compatibility?

Reply:
Are these classes still available? What would be the right structure for this
I personally don't think so because if this is the new design this is what we should want teams to do

Reply:
There are two pre-existing "CTA" stories which have a different HTML structure. We need to make sure these don't break:

https://ebay.github.io/skin/storybook/?path=/story/section-title-section-title--arrow-cta-see-all
https://ebay.github.io/skin/storybook/?path=/story/section-title-section-title--arrow-cta-no-text

Reply:
I readded it back, however the no-text variant does not exist (and we can't get it unless we add a new argument)